trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/edit-payment-helper.ts

```diff
@@ -92,6 +97,7 @@ export const getEditPaymentViewModelWithFormValues = (
   isCheckboxChecked: (checkboxId: EditPaymentsTableCheckboxId) => boolean,
   formValues: EditPaymentFormValues,
   errors: EditPaymentErrorsViewModel = EMPTY_PAYMENT_ERRORS_VIEW_MODEL,
+  redirectTab: ReconciliationForReportTab | undefined = undefined,
```

Comment:
Do we need `| undefined = undefined`? Could it just be this?:

```ts
redirectTab?: ReconciliationForReportTab
```

Reply:
Updated to match that pattern thanks, I've checked over the other touched files and there don't seem to be any other occurrences